.syncpackrc

```diff
@@ -273,7 +273,7 @@
 			"packages": [
 				"**"
 			],
-			"pinVersion": "wp-6.6"
+			"pinVersion": "^1.15.0"
```

Comment:
Thanks! We also can drop this line: https://github.com/woocommerce/woocommerce/blob/trunk/.syncpackrc#L54

Reply:
@alexflorisca: I don't think so - if we keep the line 276 as `"pinVersion": "^1.15.0"` and drop line 54 `"@wordpress/e2e-test-utils-playwright",` we just ensure that syncpack applies version `^1.15.0` to all packages.

The dependency is used in packages `@woocommerce/plugin-woocommerce` and `@woocommerce/block-library` under the `devDependencies` section, so I can not find occurrences where it faces end users.

Reply:
Yes, you're right, I was looking at the wrong line. It's done.